modules/cms/classes/Controller.php

```diff
@@ -1436,31 +1436,52 @@ public function param($name, $default = null)
      * @param array  $properties  Component properties
      * @param bool   $addToLayout Add to layout, instead of page
      * @return ComponentBase Component object
```

Comment:
@DanHarrin Would you be able to indicate in the docblock that this method can return `null` as well if a soft component is used but not found?

Reply:
Done @bennothommo 👍

Reply:
@DanHarrin Docblocks should only have one `@return` tag. I would suggest the following instead: 
```
@return ComponentBase|null Component object. Will return `null` if a soft component is used but not found.
```